.github/workflows/e2e-tests.yml

```diff
@@ -114,6 +114,26 @@ jobs:
               mv src/Baz.php.orig src/Baz.php
               echo -n > phpstan-baseline.neon
               ../../bin/phpstan -vvv
+          - script: |
+              cd e2e/result-cache-6
+              echo -n > phpstan-baseline.neon
+              ../../bin/phpstan -vvv
+              patch -b src/Baz.php < patch-1.patch
+              cat baseline-1.neon > phpstan-baseline.neon
+              ../../bin/phpstan -vvv
+              mv src/Baz.php.orig src/Baz.php
+              echo -n > phpstan-baseline.neon
+              ../../bin/phpstan -vvv
+          - script: |
+              cd e2e/result-cache-8
```

Comment:
Where did 7 go? :)

Reply:
I dropped it after I realized I need just one test for each annotation :-)